ports/tinynpy/portfile.cmake

```
@@ -1,23 +1,25 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO cdcseacave/TinyNPY
-    REF 0a11001009cc3e8f3c95c61b92c0ce1bd52d5383
-    SHA512 a5fe962e433896a5c7363808e87fca1297c98496cf6b6fd98154f4f4bc113ea213e281ebcdbceadf47fc573b6d707097f3a06cc58bf26351ebf7f935954d79c0
+    REF "v${VERSION}"
+    SHA512 3510c1da36a6260e0e3d0a2f0c4c9c12b673574a221ea6e217e5ed2c1c14d4f2937ef00dd9bd8407e52e3961821ccf1301c74691ac562f242c9b80bc8ddd4314
     HEAD_REF master
 )
 
-string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" BUILD_STATIC_LIBS)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" _BUILD_SHARED_LIBS)
+string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" LINK_CRT_STATIC)
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
-        -DBUILD_STATIC_LIBS=${BUILD_STATIC_LIBS}
-        -DBUILD_DEMO=OFF
+    -DBUILD_SHARED_LIBS=${_BUILD_SHARED_LIBS}
+    -DLINK_CRT_STATIC_LIBS=${LINK_CRT_STATIC}
+    -DBUILD_DEMO=OFF
```

```suggestion
        -DLINK_CRT_STATIC_LIBS=${LINK_CRT_STATIC}
        -DBUILD_DEMO=OFF
```
BUILD_SHARED_LIBS is handled by `vcpkg_cmake_configure`.
And please keep the indent.